content/en/docs/v3.4/op-guide/configuration.md

```diff
@@ -209,6 +209,11 @@ The list of flags provided below may not be up-to-date due to ongoing developmen
 + default: false
 + env variable: ETCD_ENABLE_V2
 
+### --next-cluster-version-compatible
++ Enable 3.4 to be compatible with next version 3.5, to allow 3.4 server to join 3.5 cluster and start on 3.5 schema.
```

Comment:
Suggest we mention downgrade as this is the main reason why this flag would be used.

```suggestion
+ Enable a 3.4 server to be compatible with version 3.5, allowing the 3.4 server to join a 3.5 cluster and start on 3.5 schema for cluster downgrade purposes.
```

Reply:
Sounds good!

Reply:
Thanks! - Would you mind please squashing commits?